packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -653,52 +674,60 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
    * @param params.globalPassword - The latest global password.
    * @returns A promise that resolves to the password corresponding to the current authPubKey in state.
```

Comment:
function docs needs to be updated.

Reply:
done in 572dad08111083fe06f632ee2fef0ca6bb206548